docs/ref-manual/cvl/types.md

```diff
@@ -89,6 +92,94 @@ multi-dimensional arrays
 reference types
 ```
 
+Types Defined in Solidity Contracts
+-----------------------------------
+
+Solidity supports several user defined types:
+* Enums
+* User Defined Value Types (type aliases)
+* Structs
+
+These types can be accessed by referencing the contract where they are defined. To reference a contract remember to include `using ContractName as myLocalContractName`. The file containing the contract `ContractName` will have to be included in your build.
```

Comment:
Just out of curiousity, do types defined in `currentContract` need to be fully qualified?